packs/jenkins/README.md

```diff
@@ -5,6 +5,7 @@ This pack creates a basic integration with Jenkins
 To trigger events from Jenkins, use Jenkins to send a webhook to
 StackStorm. Examples of rules can be found in the `rules` directory.
 
+Ensure your jenkins jobs are parameterised to avoid errors 
```

Comment:
Some additional clarification on that would be great - I have no idea what `parameterised` means in this context.

From Slack discussion I assume it means you need to provide some required parameters, but it would be good to clarify and spell this out explicitly :)